crypto/fipsmodule/rsa/rsa.c

```diff
@@ -877,7 +877,13 @@ int RSA_flags(const RSA *rsa) { return rsa->flags; }
 int RSA_test_flags(const RSA *rsa, int flags) { return rsa->flags & flags; }
 
 int RSA_blinding_on(RSA *rsa, BN_CTX *ctx) {
-  return 1;
+  return (rsa != NULL && (rsa->flags & RSA_FLAG_NO_BLINDING) == 0) ? 1 : 0;
```

Comment:
Nit-pick: Another pair of parenthesis would make this world a little happier:
```C
  return (rsa != NULL && ((rsa->flags & RSA_FLAG_NO_BLINDING) == 0)) ? 1 : 0;
```

Reply:
done